src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => {
+  return (
+    <NavLink
+      to={to}
+      className={({ isActive }) => (
+        classNames(
+          'navbar-item',
+          'isActive',
```

Comment:
```suggestion
          'isActive',
```
I don't think this style is needed.